src/view/uniprotkb/SequenceSection.tsx

```diff
@@ -23,12 +28,30 @@ const SequenceSection: FC<{
           features={data.featuresData}
           sequence={data.sequence.value}
         />
-        {data.sequenceCaution && (
+        {data.sequenceCaution && data.sequenceCaution.length > 0 && (
           <Fragment>
             <h4>Sequence caution</h4>
             <SequenceCautionView data={data.sequenceCaution} />
           </Fragment>
         )}
+        {data.massSpectrometry && data.massSpectrometry.length > 0 && (
```

Comment:
Do we strictly need the `>0` check?

Reply:
I don't think we'd want to display the titles if these were defined but empty